radars/SRE.csv

```diff
@@ -0,0 +1,951 @@
+<h3>Pros</h3>
+<ul>
+<li>Has all of your tools in one place (project management, builds, source control)</li>
+<li>Constantly adding features</li>
+<li>User and group management easy</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>UI not as friendly in some areas</li>
+<li>Not well suited for web apis</li>
+<li>Small community compared with Github</li>
+</ul>
+<p>When decision made: 1/1/2020</p>
+<p>When decision will be revisited: 1/1/2022</p>
+"
+Octopus Deploy,Infrastructure,Divest,none,"<p><a href=""https://octopus.com/"">Octopus Deploy</a> You already have a source control system and a build server. Octopus doesn’t replace these, we turbocharge them. We allow the build server to focus on what it does best: compiling code and running tests, while Octopus takes care of deploying and promoting releases between environments, automating deployments, and automating the operations tasks that keep the software running.</p>
+<h3>Pros</h3>
+<ul>
+<li>Focused on release management</li>
+<li>Works with both cloud deployments and on premise deployments</li>
+<li>Inexpensive</li>
+<li>UI is easy for engineers and others to control releases</li>
+<li>High level dashboard view of releases</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>Requires an agent for on premise servers</li>
+<li>Configuration can get complicated very quickly</li>
+<li>Small community and very few plugins</li>
+</ul>
+<p>When decision made: 1/1/2020</p>
+<p>When decision will be revisited: 1/1/2022</p>
+"
+AWS CodePipeline,Infrastructure,Invest,budding,
+New Relic,Observability,Invest,confident,"<p><a href=""https://newrelic.com/"">New Relic</a> is an analytics package that can be used to gain insights about your application. NewRelic can be depoyed as an Application Performance Monitor (APM) or to your infrastructure for insights into how the system is performing as a whole. New Relic should be used instead of <a href="""">Datadog</a>.</p>
+<h3>Pros</h3>
+<ul>
+<li>Works across Azure and NewRelic</li>
+<li>Powerful insights and dashboarding</li>
+<li>Can be customized withing the New Relic ONE platform</li>
+<li>Extremely easy integration into applications</li>
+<li>Docker support</li>
+<li>Easy to read tracing</li>
+</ul>
+<h3>Cons</h3>
+<ul>
+<li>More expensive than some other offerings</li>
+<li>Cost prohibitive to use for logs as well</li>
+</ul>
+<p>When decision made: 5/1/2020</p>
+<p>When decision will be revisited: 5/1/2022 (under 2 year contract)</p>
+"
+BugSnag,Observability,Invest,confident,"<p><a href=""https://www.bugsnag.com/"">BugSnag</a> is an error-monitoring tool that allows your developers to identify, prioritize and replicate bugs in a time-efficient and enjoyable manner. BugSnag creates an ownership mentality with all developers by enabling them to proactively see the effects of their code. This helps them address the issues before it escalates, creating a joyful experience for both customers and engineers.</p>
```

Comment:
I've never heard of us using this so if this is right, we should be socializing this more.

Reply:
This is definitely a divest. We use Bugsnag, but it's the same tool as Sentry.io, which I think is our invest.